tutorials/preparing-the-vm.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: page
+title: Preparing the Virtual Machine
+---
+## Purpose
+This lesson describes the steps for configuring a virtual machine to run Kite example code on a Cloudera Quickstart VM.
+
+### Result
+Your VM is ready for you to install and run sample programs from the Kite SDK Examples project.
+
+## Installing the VM and Kite Examples
+
+1. Install a VirtualBox or VMWare [Cloudera QuickStart VM][getvm] version 5.2 or later.
+1. In that VM, run the following command from a terminal window. This command clones a local copy of Kite code examples you can build and run.
+
+     ```bash
+     git clone https://github.com/kite-sdk/kite-examples.git
+     ```
+
+1. If you haven't already done so, install the [Kite CLI][install-cli].
+
+[getvm]: http://www.cloudera.com/content/support/en/downloads/quickstart_vms.html
+[install-cli]:{{site.baseurl}}/Install-Kite.html
+
+## Configuring the VM
+
+Some Kite examples require Flume. If you use Cloudera Manager, Flume user impersonation is configured for you. If do not use Cloudera Manager, you must update Flume user impersonation in `core-site.xml`.
```

Comment:
I'll add a section on CM, with the 8GB memory and 2 CPU recommended configurations. This is documented elsewhere and should not be a barrier to publication of our examples, even if we choose to include it.